lib/cli.rb

```diff
@@ -0,0 +1,164 @@
+require_relative '../config/environment'
+require 'pry'
+require 'colorize'
+
+
+class CLI
+
+  
+
+  def create_user
+    puts `clear`
+    puts "Please type in your name...then press enter.. I guess..."
+    user_name = STDIN.gets.chomp.strip 
+    $user = User.find_or_create_by(name: user_name)
+  end 
+  
+  def greeting
+    puts `clear`
+    puts "Weather is breezy".colorize(:light_magenta)
+    puts 
+    puts "Weather is cool" 
+    puts 
+    puts "Don't get "+"washed".colorize(:light_magenta).italic+" away by too many choices."
+    puts 
+    puts "~ Select an option below "+"#{$user.name}".colorize(:cyan).italic+" ~"
+    puts 
+  end 
+
+  def cli_program
+    is_running = true 
+    while is_running 
+      puts "1. Search "+ "Weather".colorize(:cyan)
+      puts "2. View Search History"
+      puts "3. Delete History"
+      puts "4. Change a Search Record"
+      puts "5. Exit program"
+
+      choice = STDIN.gets.chomp 
+
+      if choice == "1"
+        puts `clear`
```

Comment:
For any of these buckets where you are handling a single case, if they go beyond 6 lines of code you should probably try and extract them out into their own functions.